src/main/java/com/janboerman/f2pstarassist/plugin/StarAssistPanel.java

```diff
@@ -131,7 +131,16 @@ private class StarAssistPanelRow extends JPanel {
 
             setToolTipText(tooltipText.toString());
 
-            final String text = "T" + star.getTier().getSize() + " W" + star.getWorld() + " " + star.getLocation();
+            final String text = this.config.callOutFormat
+
+            if(text.contains("{world}") && text.contains("{tier}") && text.contains("{location}")){ // if provided format is valid

```

Comment:
In this codebase I use spaces before and after keywords, so this should be `if (` at the beginning of the line, and at the end of the line `) {`.